guarddog/analyzer/metadata/npm/empty_information.py

```diff
@@ -0,0 +1,20 @@
+""" Empty Information Detector
+
+Detects if a package contains an empty description
+"""
+import os.path
+from typing import Optional
+
+from guarddog.analyzer.metadata.empty_information import EmptyInfoDetector
+
+MESSAGE = "This package has an empty description on PyPi"
+
+
+class NPMEmptyInfoDetector(EmptyInfoDetector):
+
+    def detect(self, package_info, path: Optional[str] = None) -> tuple[bool, str]:
+        if path is None:
```

Comment:
Any way we can encode this constraint in the type signature instead? If not, it may be worth having a quick c omment about it

Reply:
Will check :))

Reply:
seems it's not straightforward, let's keep it for later